package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "grace-shopper",
+  "version": "1.0.0",
+  "description": "An online e-commerce platform for unauthenticated guests, authenticated users, and administrative users.",
+  "main": "index.js",
+  "scripts": {
+    "webpack": "webpack",
+    "webpack-watch": "webpack -w --mode=development",
+    "postinstall": "npm run webpack",
+    "seed": "node script/seed.js",
+    "start": "node server",
+    "start-watch": "nodemon server --ignore public --ignore client",
+    "start:dev": "NODE_ENV='development' npm run webpack-watch & NODE_ENV='development' npm run start-watch",
+    "test": "NODE_ENV='test' mocha \"./test/*.spec.js\"",
+    "test:users": "NODE_ENV='test' mocha \"./test/user-route-tests.spec.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/1904-cpu-projects/kach.git"
+  },
+  "keywords": [
+    "e-commerce",
+    "shopping",
+    "cart",
+    "react",
+    "redux",
+    "sequelize",
+    "express",
+    "sessions"
+  ],
+  "author": "Hugo Campos, Katherine Peterson, Conner Stennett",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/1904-cpu-projects/kach/issues"
+  },
+  "homepage": "https://github.com/1904-cpu-projects/kach#readme",
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.6",
+    "chai": "^4.2.0",
+    "mocha": "^6.2.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-router-dom": "^5.0.1",
+    "redux-logger": "^3.0.6",
+    "supertest": "^4.0.2",
+    "supertest-session": "^4.0.0",
+    "webpack": "^4.38.0",
+    "webpack-cli": "^3.3.6"
+  },
+  "dependencies": {
+    "axios": "^0.19.0",
+    "connect-session-sequelize": "^6.0.0",
+    "dotenv": "^8.0.0",
+    "express": "^4.17.1",
+    "express-session": "^1.16.2",
+    "heroku": "^7.27.1",
+    "pg": "^7.12.0",
+    "query-string": "^6.8.2",
+    "react-redux": "^7.1.0",
+    "redux": "^4.0.4",
```

Comment:
You have redux here, and react in dev - I'm indifferent where you put FE dependencies for prod - but keep them in the same area.